.github/dependabot.yml

```diff
@@ -17,6 +17,10 @@ updates:
           - "io.quarkus.*:*"
           - "io.quarkiverse:*"
           - "io.quarkiverse.*:*"
+      # any WEB component dependencies:
+      mvnpm:
```

Comment:
Maybe also add an assignee ? And ideally a monthly interval, but I'm not sure that's doable on a per group basis...

Reply:
yeah 😔 assignee/interval can only be applied at the `package-ecosystem` level... and we can't have two same ecosystems pointing to the same directory (meaning we won't be able to have maven for web group and maven for other groups 😔)

Reply:
Right. Too bad... If the dependencies were defined in different modules, we could leverage [CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners), but as things stand... Nothing we can do, I guess.
Thanks for having a look.